src/controllers/BoardController.ts

```diff
@@ -0,0 +1,27 @@
+import { Get, HttpCode, JsonController, Res } from "routing-controllers";
+import { OpenAPI } from "routing-controllers-openapi";
+import message from "../modules/responseMessage";
+import statusCode from "../modules/statusCode";
+import util from "../modules/util";
+import { Response } from "express";
+import { BoardService } from "../services/boardService";
+
+@JsonController("/board")
+export class BoardController {
+    constructor(private boardService: BoardService) { }
+
+    @HttpCode(200)
+    @Get("")
+    @OpenAPI({
+        summary: "게시판 목록 조회",
+        description: "게시물 리스트 반환",
+        statusCode: "200"
+    })
+    public async getBoardList(
```

Comment:
```suggestion
    public async getList(
```

board 컨트롤러니까 변수명에 board가 중복되지 않아도 될 것 같습니다